src/main/java/com/github/checkstyle/regression/git/GitChange.java

```diff
@@ -19,6 +19,8 @@
 
 package com.github.checkstyle.regression.git;
 
+import org.eclipse.jgit.diff.DiffEntry;
```

Comment:
Sorry I forgot it again. :( 
So your suggestion is move `GitChange` to `data` package? I am OK with it. Or maybe one more option: create a `git.data` package and move it there. Just come up another option and I have no preference on these two. :) I would wait your opinion and do the refactor then.

There is a question that whether we should expand a `ChangeType` field in `GitChange` in production code. I think that depends on whether we would treat them the same way or different way. Expect for `DELETE` (the file is gone), should we just not care about it is a new file or a modified file? By now the default config generator does not care about it.

If we decide to not introduce `ChangeType` to `GitChange` now. Then we could split `parse` method into two: one process the repository and return a list of JGit `DiffEntry`; the other convert `DiffEntry` to `GitChange`. Then we could test the `ChangeType` by calling first method.